packages/components/tests/dummy/app/templates/components/form/base-elements.hbs

```diff
@@ -406,6 +446,40 @@
   {{! prettier-ignore-end }}
   <p class="dummy-paragraph">Renders to:</p>
   <Hds::Form::Label @controlId="control-ID">My label</Hds::Form::Label>
+  <p class="dummy-paragraph">
+    Pass in a
+    <code class="dummy-code">isRequired</code>
+    argument when the user input is required for the associated form control:
+  </p>
+  {{! prettier-ignore-start }}
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::Form::Label @controlId="control-ID" @isRequired=\{{true}}>My label</Hds::Form::Label>
+    '
+  />
+  {{! template-lint-enable no-unbalanced-curlies }}
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">Renders to:</p>
```

Comment:
[suggestion] probably, now that we have the `Required` badge, I would update the example below this (the one with "Some badge") to have the badge as `highlight` color, so they are different and it doesn't generate confusion (see all the occurrences of "Some badge" in the page).

Reply:
Updated with the suggestion – have thought about this, but hesitated as people may take it as a recommendation to use the a colored badge in label.